style.css

```diff
@@ -0,0 +1,56 @@
+body {
+    background-image: linear-gradient(
+        115deg, rgb(100,70,255,0.6),rgb(100,70,255,0.6)),
+        url("background.jpeg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    font-weight: 400;
+    background-attachment: fixed;
+    font-family: Verdana;  
+    text-align: center;
+}
+form {
+    background-color: rgb(41, 34, 71,0.7);
+    width: 60%;
+    margin: 50px auto;
+    padding: 30px 20px;
+}
+.form-control label {
+    display: block;
+    margin-bottom: 1em;
+    color: white;
+    text-align: left;
+}
+
+.form-control textarea {
+    border: 1px solid #777;
+    border-radius: 2px;
+    font-family: inherit;
+    padding: 10px;
+    display: block;
+    width: 95%;
+    height: 5em;
+}
+
```

Comment:
@Elisha-Tetteyfio You should have highlighted that the student need not leave huge lines of space between codes. It makes the code difficult to read and also untidy.